app/views/internal/articles/_individual_article.html.erb

```diff
@@ -63,158 +66,178 @@
+    <br /><br />
+    <% phrase = "#{article.path}?booster_org=#{article.organization&.slug || 'generic'}" %>
+    <b>BOOSTED IN <%= EmailMessage.where("subject LIKE ?", "%#{phrase}%").where.not(opened_at: nil).size %> EMAILS</b>
+    <br />
+    <b>BOOSTED IN
+      <%= EmailMessage.where("subject LIKE ?", "%#{phrase}%").where.not(opened_at: nil).where("sent_at > ?", 1.week.ago).size %>
+      EMAILS IN THE PAST WEEK</b>
+    <br /><br />
+    <b>BOOSTED IN
+      <%= EmailMessage.where("subject LIKE ?", "%#{phrase}%").where.not(opened_at: nil).where.not(opened_at: nil).size %>
+      OPENED EMAILS</b>
+    <br />
+    <b>BOOSTED IN
+      <%= EmailMessage.where("subject LIKE ?", "%#{phrase}%").where.not(opened_at: nil).where.not(opened_at: nil).where("sent_at > ?", 1.week.ago).size %>
+      OPENED EMAILS IN THE PAST WEEK</b>
     <% end %>
 
     <div>
-      <button class="btn btn-secondary" data-toggle="collapse" data-target="#article-<%= article.id %>-buffer-area" area-expanded="false" area-controls="article-<%= article.id %>-buffer-area">
+      <button class="btn btn-secondary" data-toggle="collapse" data-target="#article-<%= article.id %>-buffer-area"
+        area-expanded="false" area-controls="article-<%= article.id %>-buffer-area">
         Share to Buffer
       </button>
       <% if (article.decorate.cached_tag_list_array & Tag.bufferized_tags).any? %>
-        (Satellites Available)
+      (Satellites Available)
       <% end %>
     </div>
 
     <div id="article-<%= article.id %>-buffer-area" class="buffering-area-for-single-article collapse">
       <% unless params[:state]&.include?("classic") %>
-        <div class="blockquote my-2">
-          <%= article.processed_html&.html_safe %>
-        </div>
+      <div class="blockquote my-2">
+        <%= article.processed_html&.html_safe %>
+      </div>
 
-        <div>
-          <%= form_with url: "/internal/buffer_updates", html: { data: { action: "submit->article#highlightElement" } } do %>
-            <input type="hidden" name="social_channel" value="main_twitter" />
-            <input type="hidden" name="article_id" value="<%= article.id %>" />
-            <h5>Twitter MAIN</h5>
-            <div class="form-group">
-              <textarea cols="37" rows="6" wrap="hard" name="tweet" maxlength="255"><%= article.title %>
-                <% if article.user.twitter_username? %>&#013{ author: @<%= article.user.twitter_username %> } #DEVCommunity<% end %>
+      <div>
+        <%= form_with url: "/internal/buffer_updates", html: { data: { action: "submit->article#highlightElement" } } do %>
+        <input type="hidden" name="social_channel" value="main_twitter" />
+        <input type="hidden" name="article_id" value="<%= article.id %>" />
+        <h5>Twitter MAIN</h5>
+        <div class="form-group">
+          <textarea cols="37" rows="6" wrap="hard" name="tweet" maxlength="255">
+                <% concat article.title
```

Comment:
A `textarea` tag preserves indentation between different ERb tags. The [`concat`](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-concat) helper allows us to build up a string inside a single tag.

I could have added a helper function for this purpose, but I'm not a big fan of their global namespace, and the two places where we need this are slightly different.